content/ui-patterns/feature-onboarding.mdx

```diff
@@ -105,7 +108,10 @@ Treat feature onboarding as a story or a journey with a beginning, middle, and e
 
 <Flex alignItems="center" flexDirection="column" my={3}>
   <BorderBox width={9 / 12} mb="0" overflow="hidden">
-    <img src="https://user-images.githubusercontent.com/66705495/92781977-7348a100-f359-11ea-9ed1-924f4ea8567a.png" alt="Example of the success state toast component " />
+    <img
+      src="https://user-images.githubusercontent.com/66705495/92781977-7348a100-f359-11ea-9ed1-924f4ea8567a.png"
+      alt="Example of the success state toast component "
```

Comment:
We repeat the prefix "Example of" many times in our alt text. We should consider dropping this to reduce verbosity:

```suggestion
      alt="The success state toast component"
```

We should also review our alt text and ensure that any relevant context is added for screen reader users.